app/views/staff/organisations/new.html.haml

```diff
@@ -17,10 +17,10 @@
           = f.input :language_code,
                     wrapper: 'select',
                     collection: yaml_to_options("language_code"),
-                    hint: I18n.t("create_organisation.language_code.hint")
+                    hint: I18n.t("form.organisation.language_code.hint")
```

Comment:
Do you think these would be better in `config/locales/simple_form.en.yml` or is that overkill?

Reply:
That sounds like a reasonable way to escalate if this `form` grouping proves to be painful and not fix anything.